### ISSUE TITLE
onpopupopen is not called with other layers than Marker

### ISSUE THREAD

**Issue by amarant** [PaulLeCam/react-leaflet#113]
Hi,
in a layer like this
```jsx
<Circle
  center={[51.51, -0.06]}
  onpopupopen={() => {
    this.setState({circleClicks: this.state.circleClicks + 1})}
  }
  radius={400}
>
  <Popup>
    <span>{`Popup in Circle, clicks: ${this.state.circleClicks}`}</span>
  </Popup>
</Circle>
```
the onpopupopen event handler is not called, it only works with a Marker layer.
The fix https://github.com/PaulLeCam/react-leaflet/pull/112 sets the second argument of the popup constructor to the popupContainer and makes it works.

Thanks.

**Comment by PaulLeCam** [comment]
Hi,
Thanks for the fix, I added it to master, it will go in the next release.

**Comment by PaulLeCam** [comment]
Published v0.10.1.
